MMM-MBTA.js

```diff
@@ -383,30 +637,36 @@ Module.register("MMM-MBTA", {
         }
         
         // Sorts them according to ETA time
-        this.stationData.sort((a,b) => (a.preDt - b.preDt));
+        this.stationData.sort((a,b) => (a.preETA - b.preETA));
         
+        // Remove trips beyond maxTime
         if (this.config.maxTime !== 0) {
-            this.stationData = this.stationData.filter(obj => (obj.preAway <= this.config.maxTime * 60));
+            this.stationData = this.stationData.filter(obj => (obj.preETA <= this.config.maxTime * 60));
+        }
+
+        // Puts one entry back in if there are no trips within maxTime
+        if (this.stationData.length === 0) {
+            this.stationData.push({
+                routeType: "9",
+                routeId: "None",
+                tripSign: "No vehicles in your timeframe",
+                alert: "No alerts",
+                preDt: "None",
+                preArr: "None",
+                preETA: "None"
+            });
```

Comment:
I would rather check if the stationData is empty when we're rendering, rather than making an "fake" station.

It's a misrepresentation of our data definition.

Reply:
Fixed